liquidprompt

```diff
@@ -929,6 +929,42 @@ __lp_theme_zsh_complete() {
     _describe 'theme' lp_theme_list
 }
 
+# Adds as much $3 character (e.g. spaces) between $1 and $2,
+# so as to make the resulting string the same width than the current terminal.
+_lp_fill() {
+
+    local left="$1" right="$2" space="${3:-" "}"
+
+    if [[ ${#space} -gt 1 ]] ; then
+        # Only keep the first character.
+        space="${space:0:1}"
+    fi
+
+    # Merge side in header
```

Comment:
I don't understand this comment. Did it refer to something specific in the dotmatrix theme?

Reply:
Yes, some rest of former vocabulary before I switched to the (obviously) simple left/right.